src/build/validator.md

```diff
@@ -1,7 +1,32 @@
 ---
-title: Validator Setup
+title: Validator Setup version 2
```

Comment:
I would not call them version 2 as this is early stage documentation -- maybe 0.2? I always have [this](https://semver.org/) in mind when thinking about versions for anything.